systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/objects/PersistentVolumeClaimUtils.java

```diff
@@ -66,12 +67,24 @@ public static void waitUntilPVCAnnotationChange(String namespaceName, String clu
         LOGGER.info("PVC annotation has changed {}", newAnnotation.toString());
     }
 
-    public static void waitForPVCDeletion(String namespaceName, int volumesCount, JbodStorage jbodStorage, String clusterName) {
+    public static void waitForPvcDeletion(String namespaceName, String pvcName) {
+        TestUtils.waitFor("Wait for PVC deletion", Constants.POLL_INTERVAL_FOR_RESOURCE_DELETION, Duration.ofMinutes(3).toMillis(), () -> {
+            if (kubeClient(namespaceName).getPersistentVolumeClaim(namespaceName, pvcName) != null) {
```

Comment:
```suggestion
            if (kubeClient().getPersistentVolumeClaim(namespaceName, pvcName) != null) {
```
you can remove it from the `kubeClient()` itself, as it will take only the `namespaceName` from the method parameter (in `getPersistentVolumeClaim`)